Archipelago.cpp

```diff
@@ -225,6 +225,53 @@ void AP_Start() {
     }
 }
 
+void AP_Shutdown() {
+    webSocket.stop();
+
+    // Reset all states
+    init = false;
+    auth = false;
+    refused = false;
+    multiworld = true;
+    isSSL = true;
+    ssl_success = false;
+    ap_player_id = 0;
+    ap_player_name.clear();
+    ap_ip.clear();
+    ap_game.clear();
+    ap_passwd.clear();
+    ap_uuid = 0;
+    rando = std::mt19937();
```

Comment:
My compiler complains that the type of rando is actually `mt19937_64`

Reply:
This line is not needed at all, recreating the rando object is not really useful, and in this case harmful as no seed is specified.